packmanager/pack_options.go

```diff
@@ -69,6 +75,13 @@ func PackAppSourceFiles(pack bool) PackOption {
 	}
 }
 
+// PackArgs sets the arguments to be passed to the application.
+func PackArgs(args []string) PackOption {
```

Comment:
Prefer variadic function for args:
```diff
- func PackArgs(args []string) PackOption {
+ func PackArgs(args ...string) PackOption {
```
Enabling the API to be used like so `WithArgs(param1, param2, list...)` without having to use `append`.